ui/app/src/services/content.ts

```diff
@@ -1102,12 +1102,14 @@ export function deleteItems(
   }).pipe(mapTo(true));
 }
 
-export function lock(site: string, path: string): Observable<boolean> {
-  return fetchContentXML(site, path, { lock: true }).pipe(mapTo(true));
+export function lock(siteId: string, path: string): Observable<boolean>;
+export function lock(siteId: string, paths: string[]): Observable<boolean>;
+export function lock(siteId: string, paths: string[] | string): Observable<boolean> {
+  return postJSON('/studio/api/2/content/items_lock_by_path', { siteId, paths }).pipe(mapTo(true));
```

Comment:
In the case of a single path sent, the backend will complain. 
```suggestion
  return postJSON('/studio/api/2/content/items_lock_by_path', { siteId, paths: asArray(paths) }).pipe(mapTo(true));
```